jobs/nightly-samba-rpm-builds.yml

```diff
@@ -37,16 +37,13 @@
     - shell: jobs/scripts/common/bootstrap.sh $WORKSPACE/jobs/scripts/nightly-samba-builds/nightly-samba-builds.sh "ghprbPullId=${ghprbPullId} ghprbTargetBranch=${ghprbTargetBranch}"
 
     publishers:
-    # TODO: create status emails
-    #- email-ext:
-    #    attach-build-log: false
-    #    attachment: null
-    #    body: $DEFAULT_CONTENT
-    #    content-type: text
-    #    presend-script: $DEFAULT_PRESEND_SCRIPT
-    #    recipients: ci-results@gluster.org
-    #    reply-to: $DEFAULT_REPLYTO
-    #    subject: $DEFAULT_SUBJECT ($GERRIT_BRANCH on CentOS-$CENTOS_VERSION/$CENTOS_ARCH)
+    - email-ext:
+        recipients: obnox@samba.org, gd@samba.org, anoopcs@cryptolab.net
+        reply-to: $DEFAULT_REPLYTO
```

Comment:
Maybe you should include https://lists.gluster.org/mailman/listinfo/ci-results too?

Reply:
yeah, we might. let's see what these mails look like and whether we want to configure mails only for nightly runs and not for PRs before adding any list. :smiley: